index.js

```diff
@@ -1,3 +1,8 @@
 
-module.exports = require('./dist/xcomponent');
-module.exports.default = module.exports;
+if (process.env.XCOMPONENT_EXCLUDE_IE_BRIDGE) {
```

Comment:
Can we make this `process.env.XCOMPONENT_FRAME_ONLY` for consistency? (since it has ramifications beyond just IE)